Source/dom/components/SymbolMaster.js

```diff
@@ -75,6 +75,20 @@ export class SymbolMaster extends Artboard {
     )
     return true
   }
+
+  unlinkFromLibrary() {
```

Comment:
When you unlink the component. What does it happen with the children? Do they point to the library? And the overrides?

Thanks in advance! I think that this functionality will be very useful :)

Reply:
You unlink a Symbol Master si there is no overrides. The nested symbols still point to the library yes, you can unlink them independently